contxt/models/events.py

```diff
@@ -109,36 +109,16 @@ class Event(ApiObject):
     deleted_at: Optional[datetime] = None
 
 
-@dataclass
-class TriggeredEventData(ApiObject):
-    _api_fields: ClassVar = (
-        ApiField("source_id"),
-        ApiField("subchain_triggered_event_id"),
-        ApiField("trigger_start_at", data_type=Parsers.datetime),
-        ApiField("trigger_end_at", data_type=Parsers.datetime),
-    )
-
-    source_id: str
-    subchain_triggered_event_id: str
-    trigger_start_at: datetime
-    trigger_end_at: datetime
-
-
 @dataclass
 class TriggeredEvent(ApiObject):
     _api_fields: ClassVar = (
         ApiField("id"),
         ApiField("event_id", creatable=True),
-        ApiField("Event", data_type=Event, attr_key="event"),
+        ApiField("Event", data_type=Event, attr_key="event", optional=True),
         ApiField("owner_id"),
-        ApiField("Owner", data_type=dict, attr_key="owner"),
+        ApiField("Owner", data_type=dict, attr_key="owner", optional=True),
         ApiField("is_public", data_type=bool),
-        ApiField(
```

Comment:
I'm having an issue with this.
```
self.events_service.create_triggered_event(event.id, json.dumps({"message": violation}))
```
produces
```
"{\"message\": \"Added statement (id: 12605) for 2019-03-01 - 2019-04-01\"}"
```

I'd like to have it without the outside quotes and escaped characters like the other rows in the database. I'm having trouble figuring out my issue. Some stackoverflows are suggesting I'm doubly-encoding.

Reply:
i don't see that method signature here, is this coming from somewhere else? it probably is getting dumped twice

Reply:
It's coming from a consumer in a separate service / PR https://github.com/ndustrialio/urjanet-worker/pull/23